app/sources/ChooseStringEncodingWindowController.m

```diff
@@ -248,6 +248,8 @@ - (void)tableViewSelectionDidChange:(NSNotification *)__unused notification
 {
     NSInteger row = tableView.selectedRow;
     if (row == -1) {
+        NSIndexSet *indexSet = [NSIndexSet indexSetWithIndex:0];
```

Comment:
What if there are search results and the user deselects a row? I don't think the first option is appropriate then.